### ISSUE TITLE
Ideas for projects we could work on. 

### ISSUE THREAD

**Issue by sparklingrobots** [drupaldiversity/contrib#3]
Our first groups are working on: 
* a D8 theme
* phpunit test conversions in D8

We also have talked about helping with the Media API initiative.  Leave your ideas here!

**Comment by gusaus** [comment]
@sparklingrobots Per our quick discussion in Slack, here's the running list of Drupal Open Learning projects (the project learning arm of Drupal Dojo) https://github.com/DrupalOpenLearning/administration/projects/2

With goals and contributors already overlapping, we're hoping to provide potential projects along with ways to onboard contributors, collaborate, manage and sustain the programs.

**Comment by zenlan** [comment]
Not an existing project, but was chatting on Slack about COD/Cream and modules/distros that deal with conference organisation. The existing ones aren't quite what I had in mind, I was thinking of a more  content-driven type of thing, almost a wizard or checklist to build a **_good_** conf, very mvp, utilising eventbrite and other 3rd party tools, pulling in cocs and pointing to diverse mailing lists to advertise, best practice, stuff like that, not reinventing the wheel, e.g. no custom ticketing/registration. 

Just a thought, perhaps for a future project. Right now my preference would be to work on unit test conversion. I love tests, you only have to please the test runner, no annoying users or clients involved!

**Comment by sparklingrobots** [comment]
mlhess [7:29 AM] 
Hi all,  if anyone wants to help the security team migrate some private issues publicly please feel free to DM me.     The directions are here: https://www.drupal.org/drupal-security-team/security-team-procedures/making-a-public-issue-for-securitydrupalorg-issues.    It requires someone who can grok (in some instances complex issues) and write a nice summary.      If you want to help out here, please feel free to let me know!

**Comment by sparklingrobots** [comment]
@catch56  recommended: 
For core contribution, something scoped could be https://www.drupal.org/node/2607260
Drupal.org
[meta] Remove deprecated code usages
Problem/Motivation Core currently has some code that is deprecated to be removed by Drupal 9.0.0, but still has hundreds of usages in core. format_string() is an obvious example: https://api.drupal.org/api/drupal/core%21includes%21bootstrap.inc/function/format_string/8 Proposed resolution Before opening the 9.x branch, ensure that all usages of deprecated code have been removed in 8.x.x.

I just committed https://www.drupal.org/node/2870194 which has a list of deprecation notices that core itself currently triggers, so it’d be possible to post an issue with a patch removing one line from the list, then find them in core, post a new patch with them fixed - that’d be about all that’s needed unless we’ve deprecated something we still need.
